### DISCUSSION TITLE
One-level Zoom Out

### DISCUSSION THREAD

**Discussion by zcysxy** [vslinko/obsidian-zoom#28] [Ideas]
Why there is no **one-level** zoom out function, but only `zoom out the entire document`. Am I missing something?

**Comment by vslinko** [comment]
@zcysxy Hello! Thank you for your feedback.

You are correct. One-level zoom out has not been implemented yet. I think it will be implemented in the future.

**Reply by jenningsb2** [reply]
Very excited for the prospect of being able to navigate up and down the heading/block tree! The current implementation is great, but this would make keyboard navigation even better!

**Reply by preinert** [reply]
Also looking forward to this capability.  I often work on a section...then a subsection, and would love the ability to back out to the section (then to the next subsection) rather than - back out to the entire doc, then to the section, then to the next subsection.

**Comment by b1uegh0st** [comment]
Want to second this. This will make it a lot more helpful (with a keyboard shortcut).

The strong "why" for it is that Zooming is to give a controlled context. But sometimes zooming all the way in takes that usefulness away.

**Comment by kwmullet** [comment]
This is a VERY useful and helpful plugin.
For me, it would be doubly so if there were one-level zoom-out. I would use the features of this plugin to take notes in class, but frequently, the information is coming at me at such quantity, that grabbing for my mouse and changing visual context, even for a moment, would be very deleterious to my note-taking ability, so as it stands now, I have to spend time massaging my notes into the proper outline level afterward because I don't have an easy way to pop up and down a level in this plugin.

**Comment by cesar-programs** [comment]
Firstly, thank you for creating such a useful plugin for Obsidian; it's made a big difference in how I approach outlining. One feature that could make it truly complete is the one-level zoom-out function. Having enjoyed this in other software, I can attest it would make the user experience even better.

**Comment by diego898** [comment]
Yes this would be great! And make this plugin take obsidian one step closer to letting us have a dynalist like experience in Obsidian!
